src/Workspaces/Core/Portable/Simplification/SimplificationOption.cs

```diff
@@ -45,9 +46,30 @@ public static class SimplificationOptions
         public static Option<bool> AllowSimplificationToBaseType { get; } = new Option<bool>(NonPerLanguageFeatureName, "AllowSimplificationToBaseType", true);
 
         /// <summary>
-        /// This option says if we should simplify away the this. or Me. in member access expression
+        /// This option says if we should simplify away the <see langword="this"/>. or <see langword="Me"/>. in member access expressions.
         /// </summary>
-        public static PerLanguageOption<bool> QualifyMemberAccessWithThisOrMe { get; } = new PerLanguageOption<bool>(PerLanguageFeatureName, "QualifyMemberAccessWithThisOrMe", defaultValue: false);
+        [Obsolete]
+        public static PerLanguageOption<bool> QualifyMemberAccessWithThisOrMe { get; } = new PerLanguageOption<bool>(PerLanguageFeatureName, "QualifyMemberAccessWithThisOrMe", false);
```

Comment:
I really hate seeing "ThisOrMe".  It's essentially putting VB and C#isms into a language neutral layer.  Can we instead just have this be QualifyInstanceMemberAccess and QualifyStaticMemberAccess?